procurement_purchase_no_grouping/views/product_category_view.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Remove `encoding="utf-8"`

Reply:
Are you sure that this isn't needed?

Reply:
This one is totally needed in XML files. It's on Python files where you don't need encoding from Python 3

Reply:
Thanks @pedrobaeza ,
LGTM